crates/fluvio-spu-schema/Cargo.toml

```diff
@@ -23,5 +23,5 @@ serde = { version = "1.0.103", features = ['derive'] }
 static_assertions = "1.1.0"
 
 # Fluvio dependencies
-fluvio-protocol = { path = "../fluvio-protocol", version = "0.6" }
+fluvio-protocol = { path = "../fluvio-protocol", version = "0.6", features = ["derive"] }
```

Comment:
since derive is probably used by everyone, we can make `derive` default features